tests/test_storage.py

```diff
@@ -599,4 +734,4 @@ def test_get_data_success(homedir):
 def test_get_data_missing_data(homedir):
     orig_filename = 'foo.txt'
     out = get_data(homedir, orig_filename)
-    assert out == '<Content deleted>'
+    assert out == '<Not Found>'
```

Comment:
As an aside, this might be confusing to a user. Knowing something was deleted is different than seeing an error stating that it can't be found.

Reply:
I was thinking that it would be more accurate to say that a file you were looking for was not found rather than assuming it was deleted. It could be the case that the file is there but under a different name since we rename files automatically when `journalist_designation` changes or perhaps the file is in a different location if for some reason the client's home directory changes. This shouldn't happen but that was my thought process.

Reply:
Fair enough. I guess a flag for `is_deleted` would make sense if we want to have something like that.